hera_stats/plots.py

```diff
@@ -340,50 +287,40 @@ def plot_kstest(pc, ax=None, sortby=None, bins=None, method="varsum"):
 
     Parameters
     ----------
-    pc: hera_pspec.container.PSpecContainer
-        PSpecContainer containing jackknife information. Must be
-        same style as that outputted by jackknives module.
+    jkset: hera_stats.jkset.JKSet
+        The jackknife set to plot. Must have shape[0] == 1.
 
     ax: matplotlib axis, optional
         The axis on which to plot the ks test. Default: None.
 
-    sortby: int, list, or string, optional
-        Item (e.g. antenna_num) to use to sort data. Default: None.
-
-    bins: int, optional
-        If specified, bins the data before doing a ks test. Can
-        be useful for small data sets. Default: None.
-
-    method: string, optional
-        Method used to calculate z-scores for Kolmogorov-Smirnov Test.
-        Options: ["varsum", "weighted"]. Default: varsum.
+    cdf: function, optional
+        A scipy.stats cumulative distribution function. If None, uses
+        scipy.stats.norm(0, 1).cdf
     """
+
     if ax is None:
         fig = plt.figure(figsize=(8, 5))
         ax = fig.add_subplot(111)
 
     # Get ks information
-    dlys, ks, pvals = stats.kstest(pc, sortby=sortby, 
-                                   method=method, bins=bins, summary=False)
+    ks, pvals = stats.kstest(jkset, summary=False, cdf=cdf)
 
-    failfrac = sum(np.array(pvals) < np.array(ks))
+    failfrac = 100. * sum(pvals < ks)/ len(ks)
```

Comment:
Why x 100?

Reply:
Because it's converting decimal to percent.

Reply:
OK, might be worth renaming the variable to something like "failpct"?